src/components/Home/Home.js

```diff
@@ -68,7 +82,7 @@ export default class Home extends Component {
             <Autocomplete onSearch={this.handleSearch} suggestionList={projectNames} />
             <Link to={{
               pathname: "/dash",
-              state: selectedNodes
+              state: { selectedNodes: selectedNodes, startDate: startDate, endDate: endDate  }
```

Comment:
with es6 we can do not need to use key: value, where the key and the value are identical. We can just state the key, so this line can be refactored to:

state: { selectedNodes, startDate, endDate  }